Formula/pwntools.rb

```diff
@@ -21,6 +21,7 @@ class Pwntools < Formula
   end
 
   depends_on "rust" => :build
+  depends_on "capstone"
```

Comment:
I don't think this includes the python bindings

Reply:
I see